services/net/syndication/SyndicationAction.cs

```diff
@@ -148,7 +148,7 @@ private async Task ImportFeedAsync(IDataSourceIngestManager dataSource, Syndicat
     private async Task FetchContent(DataSourceModel dataSource, SyndicationItem item, Uri? link)
     {
         // Fetch content body separately if required
-        if (bool.Parse(dataSource.GetConnectionValue("fetchContent")))
```

Comment:
This will fail because `DataSource.Connection.fetchContent = undefined` in most cases.